docs/sinks/oracle.md

```diff
@@ -7,8 +7,9 @@ description: Replicate data from your sources to Oracle Database.
 
 ### Configuration
 
+To start, define a new connection to Oracle in your `dozer-config.yaml` file using the steps outlined [here](../sources/oracle.md).
 
-The following configuration block can be used in `dozer-config.yaml` to define a new Oracle sink:
+Thereby, the following configuration block can be added to define a new Oracle sink:
```

Comment:
This sentence doesn't really add any clarity. Try explaining that this sink configuration defines that it will replicate the table named `transactions` to the table by the same name in the oracle database defined in `connections` with the name `oracle`